app/models/post.rb

```diff
@@ -1,2 +1,12 @@
 class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+
```

Comment:
- [ ] The same here as well. Kindly add the association between the Post model and Like model as done for the comment. You should have `has_many :likes`after line 3